### ISSUE TITLE
#384983798------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#487]
<strong>Source:</strong>`<a id="edit-button-921a0a859b514f3684d6e1e917363e67" href="/layers/921a0a859b514f3684d6e1e917363e67/edit" aria-describedby="calcite-tooltip-aebf32e3-a791-fd0e-cbf7-a56cfba3a342">`
<strong>Element Location: </strong>#edit-button-921a0a859b514f3684d6e1e917363e67
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@6081f330
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
